pkgs/development/python-modules/uri-template/default.nix

```
@@ -0,0 +1,27 @@
+{ lib, buildPythonPackage, fetchFromGitHub }:
+
+buildPythonPackage rec {
+  pname = "uri_template";
+  version = "1.1.0";
+
+  #TODO: Figure out why Pypi doesn't host the source package
+  src = fetchFromGitHub {
+    owner = "plinss";
+    repo = pname;
+    rev = "v${version}";
+    sha512 = "sha512-LFjGJrYJDSHT1ri2A5/p30ajLBUbT+p7puxrjQ1OxR09q0xq4qYin/4ZFxXzM8Oa8c0P2TbWRoOVNLx7ajgZnw==";
+  };
+
+  postPatch = ''
+    sed -i -e 's/0.0.0/${version}/' setup.py
+  '';
+
+  pythonImportsCheck = [ "${pname}" ];
```

```suggestion
  pythonImportsCheck = [ "uri_template" ];
```

package has also tests we can run in addition to the above.